README.md

```diff
@@ -52,6 +52,14 @@ Note that you can use one or the other but not both. Chore will quit and make fu
 
 ### Tips for configuring Chore
 
+For Rails, it can be necessary to add the directory you store your jobs in to the eager loading path,
+found in application.rb. You likely need a similar approach for most apps using jobs, unless you places
+them into a directory that is already eager loaded by default. One example of this might be:
+
+```ruby
+config.eager_load_paths += File.join(config.root, "app", "jobs")
```

Comment:
But thats a rails thing, isn't it? Eager Load is always different in dev
mode?

Happy to add a note around it, but this is taken almost verbatim (We have
more than this in our "add to EL path" code, so it's a bit more involved)
from our server code here at Tapjoy.

On Wed, Dec 9, 2015 at 1:15 PM, Tanner Burson <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Tapjoy/chore/pull/14#discussion_r47128740>:
>
> > @@ -52,6 +52,14 @@ Note that you can use one or the other but not both. Chore will quit and make fu
> >
> >  ### Tips for configuring Chore
> >
> > +For Rails, it can be necessary to add the directory you store your jobs in to the eager loading path,
> > +found in application.rb. You likely need a similar approach for most apps using jobs, unless you places
> > +them into a directory that is already eager loaded by default. One example of this might be:
> > +
> > +```ruby
> > +config.eager_load_paths += File.join(config.root, "app", "jobs")
>
> Should test whether this actually works. I don't think this works as
> expected in development mode.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Tapjoy/chore/pull/14/files#r47128740>.
>